### PR TITLE
updates Gigabit T&C

### LINKED ISSUE #2475: Update Gigabit Terms & Conditions
Note: changes are highlighted in the doc linked below, however, as the changes are extensive due to a brand new grants management system, it would be advisable to cut and paste the full text in case any changes were missed in the manual highlighting process. This is a legally binding document that has been reviewed/approved by our internal legal council. 

Update 2016 Terms & Conditions to this version of the 2107 Terms & Conditions found in this document: https://docs.google.com/a/mozilla.com/document/d/1v4ORY4d3hvp1P8XH2x0b2KkxKbFcqihEIaxV2Fd-9m4/edit?usp=sharing

Thanks!

cc: @lindseyinCHA

### PR DESCRIPTION
**Pull request by hannahkane**
Fixes #2475

**Changes proposed in this pull request:**
- updates Gigabit Terms page as per [this doc](https://docs.google.com/document/d/1v4ORY4d3hvp1P8XH2x0b2KkxKbFcqihEIaxV2Fd-9m4/edit#)

### CODE REVIEW

**Comment by hannahkane** [comment]
@mmmavis - sorry to throw another one at you; Gigabit team would like this updated as soon as we can, so they can start promoting

File: pages/gigabit/Terms.jsx [hunk 1]
```diff
@@ -22,120 +22,221 @@ module.exports = React.createClass({
...
+              <ol>
+                <li>Are other organizations working toward outcomes similar to those described in this proposal? If yes, explain how your work complements that of others or fills a key gap.</li>
+              </ol>
+              <li>Organizational Fit (250 Words or Less)</li>
+              <ol>
+                <li>Describe how this project fits into your organization’s mission and goals.</li>
+              </ol>
+              <li>Internet Health</li>
+              <ol>
+                <li>With which Internet health issue(s) does your project align? How does it help to build a healthier future internet?</li>
```
Comment (mmmavis) [inline pages/gigabit/Terms.jsx]: `Internet health issue` should be hyperlinked

Comment (mmmavis) [inline pages/gigabit/Terms.jsx]: `Internet health issue` should be hyperlinked

File: pages/gigabit/Terms.jsx [hunk 2]
```diff
@@ -22,120 +22,221 @@ module.exports = React.createClass({
...
+              A <a href="https://docs.google.com/spreadsheets/d/1WxZuMHz7JlcmkcVP4i-DCx7Ulr5NvNq1pSK2MgqOVbQ/edit">project budget</a>, <a href="https://drive.google.com/open?id=1Yiam58szxLTYVVSluN8zP93LgB2Z3Ulah2ZzqknRB6c">project timeline</a>, and letters of support from key collaborators must also be submitted via mozilla.fluxx.io. Templates for the 16-week project timeline and budget are available on the Mozilla Gigabit website.
             </p>
             <p>
-            Submissions and any subsequent communications or postings associated with the Fund may not contain advertisements, commercial endorsements, or any material that is indecent, obscene, hateful, discriminatory, defamatory, slanderous, libelous, unlawful or otherwise inappropriate as determined by Sponsor in its absolute discretion. Any inappropriate Submissions or material may be disqualified by Sponsor in its absolute discretion.
+              The following organizational information must be submitted along with your project description via mozilla.fluxx.io:
+            </p>
+            <ol>
+              <li>Required of all Non-Profit Applicants</li>
```
Comment (mmmavis) [inline pages/gigabit/Terms.jsx]: I think the `Required of ...` lines should be `<p`> and not bullet points.

```jsx
<p><u>Required of all Non-Profit Applicants</u></p>
<ol>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ol>

<p><u>Required of all For-Profit Applicants</u></p>
<ol>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ol>
```

It reads better this way. What do you think? @hannahkane

**Comment by hannahkane** [comment]
Thanks, @mmmavis. I added the Internet Health issues link, changes the line items to paragraphs (agreed, that's better), and removed a line item from the Fund Reporting section. Not sure what I missed in the Legal Rights in Submission section, though. Can you clarify? Thank you!

**Comment by mmmavis** [comment]
@hannahkane 

> Not sure what I missed in the Legal Rights in Submission section, though. Can you clarify?

Ah I didn't really read through the copy carefully. I checked things mostly by simply "copying the new text and searching it on the localhost page". Anyways, I just did a side-by-side comparison and the difference turned out to be something really tiny

`By entering a Submission, you agree to these Terms and Conditions and you represent and warrant that` <--- on the doc there's a comma before "and you represent and warrant that". I can't tell if that comma is absolutely needed since English is not my thing 😂 Will let you decide on that.

**Comment by hannahkane** [comment]
good catch! Thanks, @mmmavis. 

If this looks ok to you, could you please merge and push to prod whenever you have time today? Thank you very much!

**Pull request merged**